src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -175,65 +178,53 @@ def _verify(filename, md5digest):
 
 # [dxid] is a symbolic link. Create a preauthenticated URL,
 # and download it
-def _download_symbolic_link(dxid, md5digest, project, dest_filename):
+def _download_symbolic_link(dxid, md5digest, project, dest_filename, symlink_max_tries=15):
+    print(symlink_max_tries)
+    if symlink_max_tries < 1:
```

Comment:
symlink_max_tries is currently checked in the download parser, so there is no way this code will ever be triggered. I added this code just to make sure that if somebody ever uses this, we will never get infinite loop (which would be case for symlink_max_tries == 0 if the file is for some reason 'undownloadable')